README.md

```diff
@@ -76,7 +76,7 @@ As for the rest, the interface should be pretty straightforward. If you have a q
 If everything went well, melonDS and the libraries it needs should now be in the `dist` folder.
 
 #### Static builds (without DLLs, standalone executable)
-5. Install dependencies: `pacman -S mingw-w64-x86_64-{cmake,SDL2,toolchain,qt5-static,libslirp,libarchive}`
+5. Install dependencies: `pacman -S mingw-w64-x86_64-{cmake,ninja,SDL2,toolchain,qt5-static,libslirp,libarchive}`
```

Comment:
Adding `ninja` as a dependency shouldn't be needed - the instructions use `make`.